wikipron/config.py

```diff
@@ -130,7 +130,13 @@ def _get_process_pron(
         def wrapper(pron):
             for processor in processors:
                 pron = processor(pron)
-            return pron
+            # GH-59: Skip prons that are empty, or have only stress marks or
+            # syllable boundaries. The `any()` call is much faster than
+            # re.match(r"[^ˈˌ.]", pron).
+            if any(c not in "ˈˌ." for c in pron):
```

Comment:
Use a ternary here? 

   return pron if any(...) else None

Or better yet, don't explicitly return None, do something like:

    if any(...):
        return pron
    # end of function

Reply:
I did what you see here for readability reasons, but I'm happy to simplify as well. Updating now.